src/sagas/admin.js

```diff
@@ -54,6 +82,14 @@ function sendTxWrapper({ txObject, gasPrice, gasLimit, fromAddress }) {
         // callback passed into eth.contract.send() to get the txhash
         if (err) {
           reject(err);
+          if (
+            err.message ===
+            "Returned error: Error: MetaMask Tx Signature: User denied transaction signature."
+          ) {
+            toast.error("Transaction was rejected.");
+          }
+        } else {
```

Comment:
Problem is this promise will always run resolve...

You can either:
1. move the resolve into this else statement so that it's not executed AFTER reject (i dont likt this)
2. swap the position of the if statement on top with the reject and `return reject(err)` instead so the rest of the code does not execute.

Reply:
Updated.